src/app/controllers/VendorController.ts

```diff
@@ -234,10 +204,8 @@ export interface UpdateVendorInput {
   website_link?: string;
 }
 
-export interface OrderTypesInput {
-  eat_in?: boolean;
-  take_out?: boolean;
-  delivery?: boolean;
+export interface OrderTypesInput extends VendorOrderTypes {
```

Comment:
Isn't it Partial for this? VendorOrderTypes

Reply:
VendorOrderTypes has only 3 keys. and they are 'eat_in' , 'take out' and 'delivery'.

Reply:
But previously it was optional for all three, interesting ok